packages/end-user-strings/test/build-message-spec.ts

```diff
@@ -0,0 +1,10 @@
+import { expect } from "chai";
+import { buildMessage } from "../src/api";
+
+describe("buildMessage", () => {
+  it("will build an error message for the validation", () => {
```

Comment:
`validation` is not related to buildMessage.

maybe: `will build an message given multiple arguments` ?